core/CPUTopology.cpp

```diff
@@ -68,6 +68,14 @@ olympia::CoreTopologySimple::CoreTopologySimple(){
             0,
             &factories->execute_rf
         },
+        {
```

Comment:
@knute-sifive I moved this above the lsu so it is destroyed before the lsu

Reply:
Hmmmm.... you shouldn't rely on that.  What if someone changes the order in the future?  What if the framework changes the order/destruction order (there are NO rules in Sparta that dictate that).  Future developers might get indeterministic behavior that they will have to chase down.  Best to ensure the notification is sent before any teardown occurs.